documentation/getting-started.adoc

```diff
@@ -53,6 +53,68 @@ can be used:
 [source]
 oc describe all
 
+=== Topic Controller
+
+Strimzi uses a component called the Topic Controller to manage topics in the Kafka cluster. The Topic Controller
+is deployed as a process running inside your Kubernetes or OpenShift cluster. To create a new Kafka topic, a ConfigMap
+with the related configuration (name, partitions, replication factor, ...) has to be created. Based on the information
+in that ConfigMap, the Topic Controller will create a corresponding Kafka topic in the cluster.
+
+Deleting a topic ConfigMap raises the deletion of the corresponding Kafka topic as well.
+
+Strimzi allows to deploy the Topic Controller through the Cluster Controller. The cluster ConfigMap contains the Topic
+Controller configuration for doing that.
+
+Furthermore, Strimzi contains example YAML files for deploying the Topic Controller not through the Cluster Controller as well.
+
+==== Deploying through the Cluster Controller
+
+To deploy the Topic Controller through the Cluster Controller, its configuration needs to be provided in the cluster
+ConfigMap in the `topic-controller-config` field as a JSON string.
+
+For more information on the JSON configuration format see <<topic_controller_json_config>>.
+
+Of course, such a deployment comes "out of box" from deploying the Cluster Controller as explained in <<Cluster Controller>>.
+
+==== Deploying to Kubernetes
```

Comment:
We should wrap this and the following chapter into a chapter named "Deploying standalone TC" or something similar. It should also explain why do you might want to deploy it standalone (such as with custom Kafka cluster).